src/physics/thermal_conduction.hpp

```diff
@@ -172,6 +186,24 @@ class ThermalConduction : public BasePhysics {
    */
   std::unique_ptr<mfem::Coefficient> source_;
 
+  /**
+   * @brief Density coefficient
+   *
+   */
+  std::unique_ptr<mfem::Coefficient> rho_;
+
+  /**
+   * @brief Specific heat capacity
+   *
+   */
+  std::unique_ptr<mfem::Coefficient> cp_;
+
+  /**
+   * @brief Combined mass matrix coefficient (rho * cp)
+   *
+   */
+  std::unique_ptr<mfem::Coefficient> mass_coef_;
+
```

Comment:
This is a funny quantity to name, because it appears in the mass matrix calculation, but actually has no dimension of mass in it. wikipedia says this is named "volumetic heat capacity" and associates the symbol "s" with it, but I don't know if that's better or worse than the current name.

Reply:
Yeah, I wasn't really sure what to call this, but I figured this could be interpreted as "mass matrix coefficient".